_posts/2022-05-04-lucy-interview.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Interview with Lucy Liu, scikit-learn Team Member"
+date: May 4, 2022
+categories:
+  - Team
+tags:
+  - Open Source
+featured-image: lucy_card.png
+
+postauthors:
+  - name: Reshama Shaikh
+    website: https://reshamas.github.io
+    image: reshama_shaikh.jpeg 
+  - name: Lucy Liu
+    website: https://twitter.com/lucyleeow
+    image: lucyliu.jpeg
+---
+
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+Lucy Liu joined the scikit-learn Team Team in 2020. In this interview, learn more about Lucy's journey through open source, from rstats to scikit-learn.
+
+1. __Tell us about yourself.__
```

Comment:
Typo - Triage Team?

Reply:
@lucyleeow 
I changed from "Team Team" to "Team".